src/middlewared/middlewared/etc_files/local/collectd.conf

```diff
@@ -52,6 +64,11 @@ LoadPlugin zfs_arc_v2
 LoadPlugin nfsstat
 LoadPlugin write_graphite
 
+<Plugin "rrdtool">
+    RRARows ${rra_rows}
+    RRATimespan ${rra_timespan}
+</Plugin>
```

Comment:
Have you verified this actually works? Its strange that rrdtool controls that given that rrdcached is what write the rrd files now. Arent they stumbling on each other?

Reply:
Yes, it seems these should be in rrdcached plugin https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_rrdcached